docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -244,4 +244,1050 @@ You have a general idea of what nf-core offers and you know how to run an nf-cor
+
+nf-core provides a large library of modules and subworkflows: pre-made nextflow wrappers around tools that can be installed into nextflow pipelines. They are designed to be flexible but may require additional configuration to suit different use cases. Currently, there are more than [1300 nf-core modules](https://nf-co.re/modules) and [60 nf-core subworkflows](https://nf-co.re/subworkflows) (November 2024) available. Modules and subworkflows can be listed, installed, updated, removed, and patched using nf-core tooling. 
+
+While you could develop a module for this tool independently, you can save a lot of time and effort by leveraging nf-core modules and subworkflows.
+
+Let's see which modules are there:
+
+```console
+nf-core modules list remote
+```
+
+This command lists all currently available modules, > 1300. An easier way to find them, is to go to the nf-core website and visit the modules subpage [https://nf-co.re/modules](https://nf-co.re/modules). Here you can search for modules by name or tags, find documentation for each module and which nf-core pipeline uses it:
+
+![nf-core/modules](img/nf-core-modules.png)
+
+### Install an nf-core module
+
+Now let's add another tool to the pipeline.
+
+`Seqtk` is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. Here, you will use the [`seqtk trim`](https://github.com/lh3/seqtk) command to trim FASTQ files.
+
+In your pipeline, you will add a new step that will take FASTQ files from the sample sheet as inputs and will produce trimmed fastq files that can be used as an input for other tools and version information about the seqtk tools to mix into the inputs for the MultiQC process.
+
+<!-- <figure class="excalidraw">
+    --8<-- "docs/nf_template/img/pipeline.excalidraw.svg"
+</figure> -->
+
+The `nf-core modules install` command can be used to install the `seqtk/trim` module directly from the nf-core repository:
+
+```
+nf-core modules install
+```
+
+!!!warning
+
+    You need to be in the myorg-myfirstpipeline directory when executing `nf-core modules install`
+
+You can follow the prompts to find and install the module you are interested in:
+
+```console
+? Tool name: seqtk/trim
+```
+
+Once selected, the tooling will install the module in the `modules/nf-core/` folder and suggest code that you can add to your main workflow file (`workflows/myfirstpipeline.nf`).
+
+```console
+INFO     Installing 'seqtk/trim'
+INFO     Use the following statement to include this module:
+
+include { SEQTK_TRIM } from '../modules/nf-core/seqtk/trim/main'
+```
+
+To enable reporting and reproducibility, modules and subworkflows from the nf-core repository are tracked using hashes in the `modules.json` file. When modules are installed or removed using the nf-core tooling the `modules.json` file will be automatically updated.
+
+When you open the `modules.json`, you will see an entry for each module that is currently installed from the nf-core modules repository:
+
+```console
```

Comment:
Maybe supply an actual command to view the content, or instruct folks to use the VSCode UI to open it?